python/tank/commands/setup_project_params.py

```diff
@@ -1063,58 +1039,68 @@ def resolve_storages(self):
         :returns: dictionary with storage breakdown, see example above.
         """
 
-        return_data = {}
+        # a dictionary of info to return
+        storage_info = {}
```

Comment:
This method didn't seem to have unit tests for it either before. Would be great if we could add that as well to the tests.